src/Validators/EmailElementValidator.cs

```diff
@@ -43,10 +43,10 @@ public ValidationResult Validate(Element element, Dictionary<string, dynamic> vi
                 isValid = false;
             }
 
-            return new ValidationResult{
-                    IsValid = isValid,
-                    Message = isValid ? string.Empty : $"{ element.Properties.Label} must be a valid email address"
-                }; 
+            return new ValidationResult {
+                IsValid = isValid,
+                Message = !string.IsNullOrEmpty(element.Properties.CustomValidationMessage) ? element.Properties.CustomValidationMessage : "Check the " + element.Properties.Label.ToLower() + " and try again"
```

Comment:
Looks good, could you replace "Check the " + element.Properties.Label.ToLower() + " and try again" with string interpolation